Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+changeAuthor = env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR
+changeBranch = env.ghprbSourceBranch ?: CHANGE_BRANCH
+changeTarget = env.ghprbTargetBranch ?: CHANGE_TARGET
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
+    }
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk11'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '10', numToKeepStr: '')
+        timeout(time: 120, unit: 'MINUTES')
+    }
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4g'
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    mailer.buildLogScriptPR()
+
+                    checkoutRepo('optaplanner')
```

Comment:
Can we skip this? I'm fine with relying on released OptaPlanner snapshots instead of building optaplanner repository on each optaweb-vehicle-routing PR.

Reply:
Sure, no problem. Checkout the repo and building it with skipping tests does not take so long, but this is your call.

Reply:
Removed.